config/locales/petitions.en-GB.yml

```diff
@@ -68,8 +68,8 @@ en-GB:
           other: "<span class=\"count\">%{formatted_count}</span> signatures in the last hour"
       in_your_constituency:
         html:
-          one: "%{formatted_count} signature from your constituency"
-          other: "%{formatted_count} signatures from your constituency"
+          one: "%{formatted_count} signature from "
+          other: "%{formatted_count} signatures from "
```

Comment:
Might be easier for one of us to add this though as it probably impacts a test on `signature_count`.